files/setup-systemd.sh

```diff
@@ -3,9 +3,11 @@
 # Disable systemd services that are not required to start on boot.
 
 systemctl -a | awk '{print $1}' | grep \.service | \
+    grep -v apparmor | \
```

Comment:
Ah, oh... Who needs apparmor right?

Reply:
Yeah. :stuck_out_tongue_closed_eyes: 

At this point I'm beginning to wonder if the systemd disablement is even worthwhile? I'm not sure how much it's actually gaining us. Probably something worth checking.